docs/docs/upgrade/network-api-for-apps.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Migrating Apps to the 1.5 Networking API
+---
+
+# Migrating Apps to the 1.5 Networking API
+
+When you upgrade to Marathon 1.5, your apps will also be automatically modified to conform to the new networking API. However, support for deprecated networking API fields will be dropped completely in some future release according to Marathon's official API deprecation policy [MARATHON-7165](https://jira.mesosphere.com/browse/MARATHON-7165).
+
+There are two ways to enable your apps to be modified for the networking API.
```

Comment:
wordsmith: Automatic app migration may be finely tuned as per the options described below: